pkgs/sdk-pkg/src/genie/libs/sdk/apis/iosxe/eigrp/verify.py

```diff
@@ -122,3 +122,62 @@ def verify_eigrp_neighbors(
             return set(neighbors).issubset(eigrp_neighbors)
         log.error(f"Please, provide a valid format for AS, Neighbors, ip and/or vrf")
         continue
+
+
+def verify_eigrp_router_id(
+    device,
+    router_id=None,
+    vrf="default",
+    AS_N=None,
+    ip="ipv4",
+    max_time=60,
+    check_interval=10,
+):
+    """verify EIGRP IDs for a given vrf and AS active instance for ipv4 or ipv6
+    Args:
+      device (obj): Device object
+      router_id = None (list): List of router ID to check
+      vrf = "default" (str): Name of the vrf by default set to "default"
+      AS_N = None (int) : Autonomous System
```

Comment:
need a better naming(uppercase not allowed). how about auto_sys instead of AS_N?

Reply:
Hi @lsheikal here I prefer (if it's possible!) to leave AS_N since all the APIs I added in get.py and verify.py contain the same variable AS_N

Reply:
@lsheikal I committed the suggestions

Reply:
We follow PEP8 standards, its not allowed. Kindly change in other apis too.
https://peps.python.org/pep-0008/#function-and-method-arguments